src/evaluations/data/evaluation_configs.py

```diff
@@ -88,6 +91,15 @@
     math.log(3) / x for x in [
         1, 2, 4, 10, 100, 200, 300, 400, 500, 600, 700, 800, 900, 1000]
 ] + [None]
+# When we would like to split budget over multiple queries, having delta greater
+# than zero is often helpful in allowing us to use smaller amount of noise.
+ESTIMATE_EPSILON_DELTA_VALUES = [
+    (math.log(3), 1e-5), (math.log(3), 1e-6), (math.log(3), 1e-7), (None, None)
+]
+# The number of estimate queries for which the budget will be split over.
+NUM_ESTIMATE_QUERIES_VALUES = [
+    1, 2, 4, 10, 100, 200, 300, 400, 500, 600, 700, 800, 900, 1000, 2000, 3000,
```

Comment:
Nit. Joey has taught me a format trick in Python3 so that we can improve the readability of the numbers. I.e., 1000000 (1 million) can be written as 1_000_000.

Reply:
Thanks, I've made the suggested changes.